examples/blog/config.js

```diff
@@ -133,7 +133,8 @@
                     .targetReferenceField('post_id')
                     .targetFields([
                         nga.field('created_at').label('Posted'),
-                        nga.field('body').label('Comment')
+                        nga.field('body').label('Comment'),
+                        nga.field('id').isDetailLink(true)
```

Comment:
Can you just move the id at the beginning of the table? That's more conventional! :)